clap-v3-utils/src/input_validators.rs

```diff
@@ -76,6 +80,10 @@ where
 }
 
 // Return an error if a keypair file cannot be parsed.
+#[deprecated(
+    since = "1.17.2",
+    note = "please use `SignerSourceParseBuilder::new().allow_file_path().build()` instead"
```

Comment:
```suggestion
    note = "please use `SignerSourceParserBuilder::new().allow_file_path().build()` instead"
```
Looks like there 9 more of these names that need updating. I'll let you search and replace, rather than suggesting on them all

Reply:
Fixed. Thank you 🙏